src/test/java/com/openmrs/joiners/service/PersonServiceTest.java

```diff
@@ -51,4 +51,17 @@ void savePerson() {
         assertThat(personToSave.getPersonName()).isEqualTo("John Smith");
         assertThat(personToSave.getPersonAddress()).isEqualTo("Kampala");
     }
+
+    @Test
+    void updatePerson(){
+        Person person = new Person(4, "Okwere Donald", "Washiso");
```

Comment:
@Timug-123 can the test be updated to be written in such a way that it tests whether an existing person object was actually updated?